src/dynamodb.js

```diff
@@ -126,11 +139,17 @@ exports.tableSchema = (schema) => {
   tablesSchema = cloneDeep(schema);
 };
 
-function dynamoDBTestHooks (useUniqueTables = false) {
+/**
+ * @param {boolean} useUniqueTables
+ * @param {object} params
+ * @param params.inMemory Determines whether to run the local DynamoDB instance
+ * in in-memory mode, or to persist the data to disk
+ */
+function dynamoDBTestHooks (useUniqueTables = false, { inMemory } = { inMemory: false }) {
```

Comment:
Do the TypeScript type files need to be updated to include this new argument?

Reply:
Done.